timed/settings.py

```diff
@@ -158,12 +143,12 @@ def trueish(value):
     ),
     'DEFAULT_PARSER_CLASSES': (
         'rest_framework_json_api.parsers.JSONParser',
+        'rest_framework.parsers.JSONParser',
         'rest_framework.parsers.FormParser',
         'rest_framework.parsers.MultiPartParser'
     ),
     'DEFAULT_PERMISSION_CLASSES': (
         'rest_framework.permissions.IsAuthenticated',
-        'rest_framework.permissions.DjangoModelPermissions',
```

Comment:
How are you going to handle permissions on different models after you have removed this?
All authenticated user can access all end points?